packages/coder/coder/overlay/app-readme.md

```diff
@@ -0,0 +1,16 @@
+# Coder/coder
+## Overview
+Coder provides development environments in your public and private clouds. Environments are defined with Terraform, connected through a secure high-speed tunnel, and are automatically shut down when not in use to save on costs. Coder gives engineering teams the flexibility to use the cloud for workloads that are most beneficial to them.
```

Comment:
Nit: I'd remove the "are" from "and are automatically shut down" - its awkward because the previous part of the sentence ("connected through a secure high-speed tunnel") does not have an "are".